terraform/modules/pre-auth-lambda/lambda.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_lambda_function" "lambda_user_expiry" {
+  filename      = "${path.module}/lambda-user-expiry.zip"
```

Comment:
we typically have the zip filename match the function name for clarity

Reply:
This is done.